mex/common/organigram/README.md

```diff
@@ -0,0 +1,30 @@
+Helper extractor to extract organigram data.
+
+Common use cases:
+- extract organizational units of your organization (e.g. president, departments, units)
+
+# Configuration
+
+The extractor reads data from a json file, whose path is set with the settings parameter 
+`organigram_path` (see `mex.common.settings` for further info).
+
+# Extracting data
+
+The module `organigram.extract` contains functions for data extraction.
+
+# Transforming data
+
+The module `organigram.transform` contains functions for data transformation.
+
+Use the `stableTargetId` attribute of the transformed objects to set attributes 
+requiring an `OrganizationalUnitID`.
+
+# Convenience Functions
+
+The module `organigram.extract` holds convenience functions, e.g. for building a mapping 
+from email addresses or synonyms to `ExtractedOrganizationalUnit`.
```

Comment:
the documentation looks good. 

But as in the LDAP extractor, I'm irritated about transformation logic in the organigram.extract

Reply:
Do you mean the fact that convenience functions reside in extract.py? If yes, is https://jira.rki.local/browse/MX-1479 sufficinet to address that for you?

Reply:
yes, can be merged